.github/workflows/build.yml

```
@@ -0,0 +1,87 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ main, master, develop ]
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+  tags:
```

actionlint (1.7.10)

[error] unknown Webhook event "tags". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)